src/Compilers/Core/Portable/Compilation/Compilation.cs

```diff
@@ -81,7 +81,7 @@ public abstract partial class Compilation
             }
         }
 
-        internal abstract AnalyzerDriver AnalyzerForLanguage(ImmutableArray<DiagnosticAnalyzer> analyzers, AnalyzerOptions options, Func<Exception, DiagnosticAnalyzer, bool> continueOnAnalyzerException, CancellationToken cancellationToken);
+        internal abstract AnalyzerDriver AnalyzerForLanguage(ImmutableArray<DiagnosticAnalyzer> analyzers, AnalyzerOptions options, AnalyzerManager analyzerManager, Func<Exception, DiagnosticAnalyzer, bool> continueOnAnalyzerException, CancellationToken cancellationToken);
```

Comment:
Nope, we should probably rename this type from AnalyzerManager to something else that doesn't give a notion of it being the manager for everything related to analyzers.
Yes, hopefully we won't need to pass around continueOnAnalyzerException once we move IDE driver to be event queue. Until then we need this because IDE driver needs to invoke the shared Execute*Actions methods which needs this parameter.